### PR TITLE
Feature/pan rankvariant

### PR DESCRIPTION
**Pull request by jemten**
### This PR fixes:

- adds the rankvariant recipe to the panel pipe
- adds the new suffix 'all' in tvcfparser to denote the research vcf for the panel pipe. This is done to enable use of the regular rankvariant recipe.   
- part of #1424

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

**Comment by jemten** [comment]
I thinking about rewriting this again and remove the 'all' suffix.

**Comment by henrikstranneheim** [comment]
👍 . I will wait - lots of meetings all day.....

**Comment by jemten** [comment]
After some back and forth I decided to keep the 'all' identifier. The PR is ready for review

File: lib/MIP/Recipes/Analysis/Mip_vcfparser.pm [hunk 1]
```diff
@@ -602,6 +602,7 @@ sub analysis_mip_vcfparser_panel {
 
     my @vcfparser_analysis_suffixes = get_vcf_parser_analysis_suffix(
         {
+            analysis_type           => $parameter_href->{cache}{consensus_analysis_type},
```
[the remark this answers is not in the document]
Comment (jemten) [reply lib/MIP/Recipes/Analysis/Mip_vcfparser.pm]: good point!

**Pull request merged**